src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<root>
+<>
```

Comment:
Is this change intentional?

Reply:
Hmm, I think this was a end-tag extension fix up problem when there were merge conflicts.